backend/src/pilstark/json_exporter/mod.rs

```diff
@@ -156,7 +172,7 @@ impl<'a, T: FieldElement> Exporter<'a, T> {
         }
     }
 
-    fn references(&self) -> JsonValue {
+    fn references(&self) -> HashMap<String, Reference> {
```

Comment:
Is this HashMap find wrt determinism?

Reply:
No. Every pil.json we print now is different.

Reply:
I'll make a PR to eigen-zkvm.

Reply:
https://github.com/0xEigenLabs/eigen-zkvm/pull/95